pkg/values/pb/values.pb.go

```diff
@@ -2,7 +2,7 @@
 // versions:
 // 	protoc-gen-go v1.31.0
 // 	protoc        v4.25.1
-// source: values.proto
```

Comment:
Could you uncommit these changes if you haven't actually changed anything related to values in this PR

Reply:
....I keep doing that, will sort

Reply:
sorted